### ISSUE TITLE
Linux build requires libXScrnSaver

### ISSUE THREAD

**Issue by spanicho-bnet** [Microsoft/BotFramework-Emulator#753]
You may want to note in your How to Build From Source documentation that the Linux build requires the libXScrnSaver package.
https://github.com/Microsoft/BotFramework-Emulator/blob/v4/CONTRIBUTING.md#how-to-build-from-source

If this package is not present, the following error is received upon launching the emulator with `npm run start:electron:dev`:
```
/home/stephen.panicho/Downloads/BotFramework-Emulator/node_modules/electron/dist/electron: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! @bfemulator/main@4.0.15-alpha start:electron: `electron --inspect=7777 .`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the @bfemulator/main@4.0.15-alpha start:electron script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/stephen.panicho/.npm/_logs/2018-08-08T16_25_49_997Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @bfemulator/main@4.0.15-alpha start:electron:dev: `cross-env ELECTRON_TARGET_URL=http://localhost:3000/ npm run start:electron`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @bfemulator/main@4.0.15-alpha start:electron:dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/stephen.panicho/.npm/_logs/2018-08-08T16_25_50_010Z-debug.log
```

**Comment by justinwilaby** [comment]
Thank you for the submission. We are in the process of auditing our `readme` which includes build/run instructions. I'll be sure to incorporate this.

**Comment by sgellock** [comment]
Doc a pre-preq for Linux

**Comment by tonyanziano** [comment]
@spanicho-bnet Can I ask what version / distro of Linux you were running? I just tried to build the source locally on Ubuntu 16.0.5 LTS and didn't run into this issue. Maybe that version comes with the package already installed natively?

**Comment by tonyanziano** [comment]
Addressed in #1028
